generators/cleanup.js

```diff
@@ -343,6 +343,10 @@ function cleanupOldFiles(generator) {
       generator.removeFile('.npmrc');
     }
   }
+  if (generator.isJhipsterVersionLessThan('7.8.2')) {
+    generator.removeFile(`${constants.DOCKER_DIR}realm-config/jhipster-users-0.json`);
```

Comment:
Should go here:
https://github.com/jhipster/generator-jhipster/blob/main/generators/server/cleanup.js